CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 ### Version 8.15
 * Supports PUT without a body parameter
+* Supports substitutions in @Headers like in @Body, that is with parts of the value
```

Comment:
Please take the below text and place it into the PR description and link to that. Entries are mostly one sentence which allows folks uninterested to skip easier. Ex "supports substitutions in `@Headers` like in `@Body`. See #326

Note I escaped the `@`